src/System Application/App/Email/src/Email/Outbox/EmailEditor.Codeunit.al

```diff
@@ -162,18 +162,22 @@ codeunit 8906 "Email Editor"
     var
         FileName: Text;
         Instream: InStream;
-        AttachmentName, ContentType : Text[250];
-        AttachamentSize: Integer;
     begin
         UploadIntoStream('', '', '', FileName, Instream);
-        if FileName = '' then
```

Comment:
Feel free to won't fix since it's only a single exit, but I think these exits makes the code easier to understand and especially add more exceptions to. Thinking of reasons to exit instead of reasons to execute.